ui/client/components/right-panel/SideNodeDetails.js

```diff
@@ -11,23 +10,25 @@ class SideNodeDetails extends Component {
       if (_.isArray(val)) {
```

Comment:
What a beautiful ifhell :)

Reply:
I think need there some factory to creating FlatObjectEntry.  Alsow in render method you don't need else statement.